parts/k8s/manifests/kubernetesmaster-kube-addon-manager.yaml

```diff
@@ -24,11 +27,23 @@ spec:
     - name: msi
       mountPath: /var/lib/waagent/ManagedIdentity-Settings
       readOnly: true
+    - name: var-lib-kubelet
+      mountPath: /var/lib/kubelet
+      readOnly: true
+    - name: etc-kubernetes
+      mountPath: /etc/kubernetes
+      readOnly: true
   volumes:
   - name: addons
     hostPath:
       path: /etc/kubernetes/addons/init
   - name: msi
     hostPath:
       path: /var/lib/waagent/ManagedIdentity-Settings
+  - name: var-lib-kubelet
+    hostPath:
+      path: /var/lib/kubelet
+  - name: etc-kubernetes
+    hostPath:
+      path: /etc/kubernetes
```

Comment:
You might need `type: Directory` here, unless it's smart enough to figure that out

Reply:
I was just looking at the CCM yaml, don't think we need it https://github.com/Azure/aks-engine/blob/master/parts/k8s/manifests/kubernetesmaster-cloud-controller-manager.yaml